src/util/jira-utils.test.ts

```diff
@@ -126,6 +125,13 @@ describe("Jira Utils", () => {
 		it("should extract multiple issue keys in a single string", () => {
 			expect(jiraIssueKeyParser("JRA-123 Jra-456-jra-901\n[bah-321]")).toEqual(["JRA-123", "JRA-456", "JRA-901", "BAH-321"]);
 		});
+
+		it("should not extract issue keys longer than 256 characters for project key or number", () => {
```

Comment:
Can you please clarify for `issue keys longer than 256 characters` in case of `ABC-123`, is it the whole string cannot be more than 256 or just each part (`ABC` and `123`) cannot be more than 256?